bedrock/products/templates/products/vpn/platforms/desktop.html

```diff
@@ -0,0 +1,173 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import hero_image, join_waitlist_button, vpn_nav_platform_cta, vpn_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_title_full %}Mozilla VPN: Protect Your Entire Device{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}Use the Mozilla VPN for full-device protection for all apps. With servers in 30+ countries, you can connect to anywhere, from anywhere.{% endblock %}
+
+{% block body_id %}mozilla-vpn-landing{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-product-page' %}
```

Comment:
Do we need a unique utm_source for this page?

Reply:
Yes – I'm setting it as `www.mozilla.org-vpn-desktop-page`, switching out "desktop" for each page. Does that work?